### PR TITLE
Key Manager Redesign

### PR DESCRIPTION
**Pull request by fhildeb**
This PR introduces **updated headings, notes, links, and description**

### Key Manager

- Added permission groups and external links for each
- Updated the nonce retrieval with LSP25 notice
- Styled permission columns

### Screenshots
Bildschirmfoto 2023-11-14 um 11 52 20
Bildschirmfoto 2023-11-20 um 14 03 54

### CODE REVIEW

File: components/KeyManagerNonceChecker/KeyManagerNonceChecker.tsx [hunk 1]
```diff
@@ -47,18 +49,79 @@ const KeyManagerNonceChecker: React.FC = () => {
...
-        Manager.
-      </p>
+      <article className="message is-info">
+        <div className="message-body">
+          This tool will retrieve the nonce of a
+          <a
+            href="https://docs.lukso.tech/standards/universal-profile/lsp0-erc725account"
+            target="_blank"
+            rel="noreferrer"
+            className="ml-1"
```
Comment (Hugoo) [inline components/KeyManagerNonceChecker/KeyManagerNonceChecker.tsx:59]: Why does any single link a tag needs a custom ml or mt ? It sounds off to me.

Not a big deal but I don't understand why

File: components/KeyManagerPermissions/KeyManagerPermissions.tsx [hunk 2]
```diff
@@ -39,20 +39,57 @@ const KeyManagerPermissions: React.FC = () => {
...
+            href="https://docs.lukso.tech/standards/universal-profile/lsp6-key-manager"
+            target="_blank"
+            rel="noreferrer"
+            className="mx-1"
+          >
+            LSP6 KeyManager
+          </a>
+          standardization to match them with their
+          <a
+            href="https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-2-ERC725YJSONSchema.md"
```
Comment (Hugoo) [inline components/KeyManagerPermissions/KeyManagerPermissions.tsx:62]: Let's update to

https://docs.lukso.tech/standards/generic-standards/lsp2-json-schema

File: components/KeyManagerPermissions/KeyManagerPermissions.tsx [hunk 3]
```diff
@@ -39,20 +39,57 @@ const KeyManagerPermissions: React.FC = () => {
...
+            href="https://docs.lukso.tech/tools/erc725js/classes/ERC725#encodepermissions"
+            target="_blank"
+            rel="noreferrer"
+            className="mx-1"
+          >
+            encodePermissions
+          </a>
+          function of the
+          <a
+            href="https://www.npmjs.com/package/@erc725/erc725.js"
```
Comment (Hugoo) [inline components/KeyManagerPermissions/KeyManagerPermissions.tsx:85]: Can you use the getting started page of erc725js page on our docs instead? (Cf other comment in other PR)

File: components/KeyManagerNonceChecker/KeyManagerNonceChecker.tsx (hunk 1 above)
Comment (fhildeb) [reply components/KeyManagerNonceChecker/KeyManagerNonceChecker.tsx:59]: Same as on the other repository:

Its either `mx-1` on every **mid-sencente**-link or `{' '}` behind AND in front of the element. Class looks much nicer and failsafe imo. Also saw it in the examples of Bulma.

**Comment by Hugoo** [comment]
Let's stick to previous text. I think "This tool xxx", "This tool yyy" just add useless text to the page. Also, do we really need callout everywhere? Less is more. Ill fix.

File: components/KeyManagerNonceChecker/KeyManagerNonceChecker.tsx [hunk 4]
```diff
@@ -47,18 +49,79 @@ const KeyManagerNonceChecker: React.FC = () => {
...
+            className="mx-1"
+          >
+            LSP6 KeyManager
+          </a>
+          smart contract.
+        </div>
+      </article>
+      <article className="message">
+        <div className="message-body">
+          It`s calling the
```
Comment (Hugoo) [inline components/KeyManagerNonceChecker/KeyManagerNonceChecker.tsx:86]:
```suggestion
          It's calling the
```

File: components/KeyManagerPermissions/KeyManagerPermissions.tsx [hunk 5]
```diff
@@ -68,19 +105,52 @@ const KeyManagerPermissions: React.FC = () => {
         </div>
       </div>
       <div className="columns">
-        <div className="column">
+        <div className="column is-half">
+          <h2 className="mb-2 subtitle is-5">
```
Comment (Hugoo) [inline components/KeyManagerPermissions/KeyManagerPermissions.tsx:109]: is it a h2 or a h5? lets not mix and match.Ill fix

File: components/KeyManagerPermissions/KeyManagerPermissions.tsx [hunk 6]
```diff
@@ -68,19 +105,52 @@ const KeyManagerPermissions: React.FC = () => {
...
           <PermissionsBtns
             permissions={['ADDEXTENSIONS', 'CHANGEEXTENSIONS']}
             color={'is-orange'}
             decodedPermissions={decodedPermissions}
             handlePermissionClick={handlePermissionClick}
           />
+          <h2 className="mb-2 subtitle is-5">
+            Manage Universal Receivers
+            <a
+              href="https://docs.lukso.tech/standards/universal-profile/lsp6-key-manager/#permissions"
```
Comment (Hugoo) [inline components/KeyManagerPermissions/KeyManagerPermissions.tsx:146]: what's the point of having all of these arrow going to the same link? That's confusing to me. I'll simply remove as it adds no value. There is already a link at the top of that page.

File: components/KeyManagerNonceChecker/KeyManagerNonceChecker.tsx (hunk 1 above)
Comment (Hugoo) [reply components/KeyManagerNonceChecker/KeyManagerNonceChecker.tsx:61]: I reco to stick with `{' '}` let's not create fake spaces by using margin. Let's try to stick to a good semantic.

File: components/KeyManagerPermissions/KeyManagerPermissions.tsx (hunk 3 above)
Comment (Hugoo) [reply components/KeyManagerPermissions/KeyManagerPermissions.tsx:72]: cf other pr - ok

**Review by Hugoo — APPROVED** [review]

**Pull request merged**